keywords.txt

```diff
@@ -0,0 +1 @@
+freeMemory				KEYWORD2
```

Comment:
This does not have the correct formatting. See the Arduino library specification for details:
https://arduino.github.io/arduino-cli/latest/library-specification/#keywordstxt-format

Reply:
I was following a example from another library - have corrected

Reply:
I have fixed this in line with the standard